src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -520,9 +538,16 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
         newDevice.stream = stream;
         newDevice.constraints = proposedConstraints;
       } else {
+        //Check can be removed after we confirm that getUserMedia bug has been fixed by chromium
+        if (
+          kind === 'audio' &&
+          this.activeDevices[kind] &&
```

Comment:
can we just check if the parameter device === default here?

Reply:
We dont know how the change in the video input will behave, if lets say we connect a usb webcam in the middle of the meeting session this device will become the default.
Also since we are targeting AudioTrack only, we are checking on 'audio' kind

Reply:
I meant we should check if device === default here too because the bug only happens if we switch from default to default right?